source/agora/consensus/validation/Block.d

```diff
@@ -188,6 +190,9 @@ public string isGenesisBlockInvalidReason (const ref Block block) nothrow @safe
         if (tx.outputs.length == 0)
             return "GenesisBlock: No output(s) in the transaction";
 
+        if (tx.payload.data.length != 0)
+            return "GenesisBlock: The data payload cannot be stored";
```

Comment:
This line is not tested according to https://codecov.io/gh/bpfkorea/agora/pull/1368/src/source/agora/consensus/validation/Block.d.

Please make sure to always add a test when adding new consensus rules. :)